docs/zh_CN/rules/overview.md

```diff
@@ -158,8 +158,14 @@
 
 ### 模版中支持的函数
 
+用户可通过模板函数，对数据进行各种转换，包括但不限于格式转换，数学计算和编码等。Kuiper 中支持的模板函数包括以下几类：
+
+1. Go 语言内置[模板函数](https://golang.org/pkg/text/template/#hdr-Functions)。
+2. 来自 [sprig library](http://masterminds.github.io/sprig/) 的丰富的扩展函数集。
+3. Kuiper 扩展的函数。
+
 Kuiper 扩展了几个可以在模版中使用的函数。
 
-- `json para1`: `json` 函数用于将 map 内容转换为 JSON 字符串
-- `base64 para1`: `base64` 函数用于将参数值编码为 base64 字符串
-- `add para1 para2`: `add` 函数用于将两个数值类型的参数相加
```

Comment:
Where is this func? Is it deprecated?

Reply:
sprig library provides a func with the same name. So the user can continue to use it. Internally, we just delete our implementation